ADOPTERS.md

```diff
@@ -0,0 +1,33 @@
+## containerd Adopters
+
+A non-exhaustive list of containerd adopters is provided below.
+
+**_Docker/Moby engine_** - Containerd began life prior to its CNCF adoption as a lower-layer
+runtime manager for `runc` processes below the Docker engine. Continuing today, containerd
+has extremely broad production usage as a component of the Docker engine stack. Note that this
+includes any use of the open source Moby engine project; including the Balena project
+listed below.
+
+**_IBM Cloud Kubernetes Service (IKS)_** - offers containerd as the CRI runtime for v1.11 and, soon, v1.12.
```

Comment:
link..

Reply:
suggest removing version data beyond just saying vx.x+ as that will need require frequent updates..